lib/internal/http2/core.js

```diff
@@ -128,8 +128,7 @@ const fsPromisesInternal = require('internal/fs/promises');
 const { utcDate } = require('internal/http');
 const { onServerStream,
         Http2ServerRequest,
-        Http2ServerResponse,
-} = require('internal/http2/compat');
+        Http2ServerResponse } = require('internal/http2/compat');
```

Comment:
I think the style used below would be a better change, specifically moving `onServerStream,` to the next line instead of this line change. Ditto for similar situations in other files as well.

Reply:
Same comment as above: I agree. Unfortunately this is the way it auto-fixes these things. I can go through and manually adjust them, though...tedious, but probably worth it to keep the code a bit more readable.